src/AgentDefinitions.H

```diff
@@ -76,24 +79,41 @@ struct IntIdxDisease
 struct SchoolType
 {
     enum {
-        total = 0,   /*!< All School type */
-        high,    /*!< High School */
-        middle,      /*!< Middle School */
-        elem_3,      /*!< Elementary School Neighbordhood 1 & 2 */
-        elem_4,      /*!< Elementary School Neighbordhood 3 & 4 */
-        day_care,     /*!< Day care + Playground */
-        total_school_type
+        none,
+        college,
+        high,
+        middle,
+        elem,
+        day_care,
+        total
     };
 };
 
+/*! \brief School types used only in initializing census data approach */
+struct SchoolCensusIDType
+{
+    enum {
+        none,
+        high_1,         /*!< High School */
+        middle_2,       /*!< Middle School */
+        elem_3,         /*!< Elementary School, nborhoods 1 and 2 */
+        elem_4,         /*!< Elementary School, nborhoods 3 and 4 */
+        day_care_5,     /*!< Day care + Playground */
```

Comment:
Why did you append the index to the name?

Reply:
Because there are actually two sets of enums, one for the school levels in general (`SchoolType`), and one specific to the census data initialization (`SchoolCensusIDType'), which has two levels of elementary school and is only used when setting up the census data. I appended the counter to make it clear when this latter type was being used. Having said that, this is rather ugly and if there is some way of avoiding having this additional census specific enum, that would be a good thing.